### PR TITLE
docs: wakeword blog post

### PR DESCRIPTION
**Pull request by noelweichbrodt**
### PR Checklist

Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [x] I am requesting to **pull a topic/feature/bugfix branch** (right side). In other words, not _master_.
- [x] I have run `npm test` against my changes and tests pass.
- [x] I have added or edited necessary documentation, or no docs changes are needed.

### Description

Adds a new blog post about wakewords

**Fixes**: #

### CODE REVIEW

File: content/blog/which-wakeword-would-we-want/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Spokestack offers two wakeword services in iOS: `appleWakeword` and `tfLiteWakeword`. Which should you use?
+
+The answer, of course, is up to you! Spokestack always gives you options, because one size does not fit everyone, and because, like you, we hate vendor lock-in.
+
+## Wait, there's a wakeword?
+
+Indeed! If you want your app to be controllable purely by voice, you need a wakeword — a word (or short phrase) that tells your app “the next thing the user says is meant for you”. The wakeword detection component in Spokestack is responsible for detecting utterances of any of a set of keyword phrases in [soft real time](https://en.wikipedia.org/wiki/Real-time_computing#Criteria_for_real-time_computing). Once detected, the Spokestack pipeline activates, providing you with an activation event and triggering the configured speech recognition service. The accuracy, speed, and flexibility of wakeword detection depends on which service you configure.
```
Comment (space-pope) [inline content/blog/which-wakeword-would-we-want/index.md]:
```suggestion
Indeed! If you want your app to be controllable purely by voice, you need a wakeword — a word (or short phrase) that tells your app “the next thing the user says is meant for you”. The wakeword detection component in Spokestack is responsible for detecting any of a user-defined set of keyword phrases in [soft real time](https://en.wikipedia.org/wiki/Real-time_computing#Criteria_for_real-time_computing). Once detected, the Spokestack pipeline activates, providing you with an activation event and triggering the configured speech recognition service. The accuracy, speed, and flexibility of wakeword detection depends on which service you configure.
```

File: content/blog/which-wakeword-would-we-want/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+### Avoid
+
+`appleWakeword` should not be used in any published app, both because its performance will always be slower than `tfLiteWakeword` and because it depends on running constant ASR over all speech while the pipeline is running, which is both a privacy concern and an overuse of resources for the actual task.
+
+## How do I distribute my app with a fast, accurate, efficient, on-device wakeword?
+
+Spokestack's `tfLiteWakeword` fits the bill for you! You'll gain fast, accurate, efficient, wakeword activation that runs entirely on-device. It features a state of the art machine learning pipeline using attention-based models; they operate continuously, each feeding output into the next, for both efficiency and accuracy.
```
Comment (space-pope) [inline content/blog/which-wakeword-would-we-want/index.md]:
```suggestion
Spokestack's `tfLiteWakeword` fits the bill for you! You'll gain fast, accurate, efficient, wakeword activation that runs entirely on-device. It features a state-of-the-art machine learning pipeline using attention-based models; they operate continuously, each feeding output into the next, for both efficiency and accuracy.
```
Phrases like this should typically be hyphenated when used as an adjective and unhyphenated when used as a noun.

**Review by space-pope — COMMENTED** [review]
Just a couple mostly optional suggestions. Good stuff!

File: content/blog/which-wakeword-would-we-want/index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Spokestack offers two wakeword services in iOS: `appleWakeword` and `tfLiteWakeword`. Which should you use?
```
Comment (timmywil) [inline content/blog/which-wakeword-would-we-want/index.md]: The main header is automatically added based on the title in the frontmatter. Here's my suggestion...

```suggestion
---
title: Choosing the Right Wakeword Service in iOS
date: '2020-06-12'
description: Spokestack offers two wakeword services in iOS: `appleWakeword` and `tfLiteWakeword`. Which should you use?
author: noel
tags: Engineering
draft: false
---

Spokestack offers two wakeword services in iOS: `appleWakeword` and `tfLiteWakeword`. Which should you use?
```

**Review by timmywil — CHANGES_REQUESTED** [review]

File: content/blog/which-wakeword-would-we-want/index.md (hunk 3 above)
Comment (timmywil) [reply content/blog/which-wakeword-would-we-want/index.md]: I'd also change the filename to match the title (all lowercase and spaces replaced with `-`)

**Review by timmywil — APPROVED** [review]

**Pull request merged**